### DISCUSSION TITLE
Logo link?

### DISCUSSION THREAD

**Discussion by gepopp** [laravel/nova-issues#4088] [Q&A]
- Laravel Version: 9.1
- Nova Version: 4.0
- PHP Version: 8.1

In v3 with the publishable blade files the link on the logo in the sidebar could easaly be overridden.
This seems to be not possible now in v4 enymore?

Would be nice to be able to set a link to the frontend of the site linking the main dashboard there doesnt make to much sense from my point of view.

**Comment by schonhoff** [comment]
Had the same problem. I added a link on the menu. I'm still waiting to modify the vue components because of the footer.
See https://github.com/laravel/nova-issues/discussions/3892#discussioncomment-2531336

**Comment by gepopp** [comment]
one month still no reply?

looks like a "buy and shush" customer care policy from @taylorotwell s team....

**Reply by crynobone** [reply]
You posted this under **Idea & Feature Requests**, we don't always respond that we still not working on this at the moment.

SVG supports adding `<a href>`, you can do that right now https://developer.mozilla.org/en-US/docs/Web/SVG/Element/a

**Reply by gepopp** [reply]
I See, again a workarround for a not thought through product.

**Reply by davidhemphill** [reply]
Would you like a refund? Would make a special exception for you since you seem so displeased...

**Reply by gepopp** [reply]
No i dont! At the end i still think its a great peace of Software u created. But These small Details show your Lack of practical expirience and i rely on that u gonne fix them. Its like some splinters in a great peace of cake😑

Putting a anchor Tag in an svg is not the issue, but for that then i have to have a second Logo file, thats how rhings start to ge messe, isnt it?

**Reply by gepopp** [reply]
@crynobone to put an anchor inside the svg is not working since the svg is surrounded by a div which catches click events, i guess its an inertia link... or smth

**Comment by Pondake** [comment]
Would love to see this implemented. Branding has gotten a big downgrade since v3 which I don't mind, but for most customer journeys you'd want the front-end application to be linked to the logo. This currently seems impossible.

Putting the anchor inside the SVG is not a solution.

**Reply by RhysLees** [reply]
You can always add a menu item for the user dropdown

**Reply by schonhoff** [reply]
I did this but it opens an extra tab instead using the same tab because you need to use
```php
MenuItem::externalLink(__('Back to frontend'), route('home')),
```
It is not a nice UX in my opinion.

**Reply by gepopp** [reply]
In fact it isnt a good UX, its simply a habbit to click on the logo in the top left corner to open the homepage... 
I still dont see why the devs decline such a widely common feature.

**Reply by RhysLees** [reply]
> In fact it isnt a good UX, its simply a habbit to click on the logo in the top left corner to open the homepage... I still dont see why the devs decline such a widely common feature.

I agree, I think it should allow us to use the logo to go to homepage, I was simply suggesting a workaround for now.

**Comment by Illizian** [comment]
Hey peeps, can we please get this prioritised! I've been waiting for this feature for over a year, and my client is absolutely loosing the plot now 🤣 

Cc. @davidhemphill

**Reply by davidhemphill** [reply]
Hi @Illizian, I've not been on the Nova team since February, and can't really help in this regard any more!

**Reply by davidhemphill** [reply]
Hi @Illizian, I've not been on the Nova team since February, and can't really help in this regard any more!

**Reply by Illizian** [reply]
Thanks @davidhemphill, didn't realise. Who's best placed for these requests now?

**Comment by jeremynikolic** [comment]
@Illizian Thanks for reaching out to us and your patience, it's been some time 🙏 

Having the possibility to customize the logo's URL would indeed be nice imo
I've noted this for me to look into 👍

**Comment by simonvomeyser** [comment]
It's a little hacky, but since I needed that feature too and had to find a way myself, here is what I did:

You can add custom JavaScript to your Nova backend. In your NovaServiceProvider:

```php
// NovaServiceProvider.php - boot method
Nova::script('custom', public_path('js/nova-custom.js'));
```

And in that script, simply catch the click and change the location.

```js
// custom.js
Nova.booted(() => {
    const element = document.getElementById('your-logo'); // if you use an svg, give it that id in the <svg ...> tag
    element?.addEventListener('click', (e) => {
        e.preventDefault();
        window.location.href = '/';
    });
});
```
